language-server/src/server.js

```diff
@@ -355,5 +360,35 @@ connection.languages.semanticTokens.onDelta(async ({ textDocument, previousResul
   return builder.buildEdits();
 });
 
+// $SCHEMA COMPLETION
+connection.onCompletion((textDocumentPosition) => {
+  const doc = documents.get(textDocumentPosition.textDocument.uri);
+  if (!doc) {
+    return [];
+  }
+
+  const instance = JsoncInstance.fromTextDocument(doc);
+
+  const currentProperty = instance.getInstanceAtPosition(textDocumentPosition.position);
+  if (currentProperty.pointer.endsWith("/$schema")) {
+    return getDialectIds().map((uri) => {
+      const range = {
+        start: currentProperty.startPosition(),
+        end: {
+          line: currentProperty.endPosition().line,
+          character: currentProperty.endPosition().character + 1
+        }
+      };
+      return {
+        label: `"${uri}"`,
+        kind: CompletionItemKind.Keyword,
```

Comment:
I don't think `Keyword` is the right thing here. `$schema` is the keyword, but that's not the thing that's getting completed. It's the value of `$schema` that's getting completed. I'm not sure what the best choice is, but it's not `Keyword`.

Reply:
Yeah, `Value` or `Property` can be used instead of `Keyword`.